util/slack.go

```diff
@@ -0,0 +1,49 @@
+package util
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/lbryio/lbry.go/errors"
+	"github.com/nlopes/slack"
+	log "github.com/sirupsen/logrus"
+)
+
+const slackChannel = "youtube-status"
+
+var slackApi *slack.Client
+
+func InitSlack(token string) {
+	slackApi = slack.New(token)
+}
+func SendToSlackInfo(format string, a ...interface{}) {
+	message := fmt.Sprintf(format, a...)
+	if slackApi != nil {
+		sendToSlack(":information_source: " + message)
+	}
+	log.Debugln(message)
+}
+func SendToSlackError(format string, a ...interface{}) {
+	message := fmt.Sprintf(format, a...)
+	if slackApi != nil {
+		sendToSlack(":sos: " + message)
+	}
+	log.Errorln(message)
+}
+
+func sendToSlack(message string) {
+	var err error
+	if slackApi == nil {
+		err = errors.Err("no slack token provided")
+	} else {
+		host, err := os.Hostname()
+		if err != nil {
+			host = "???"
+		}
+		_, _, err = slackApi.PostMessage(slackChannel, message, slack.PostMessageParameters{Username: "YTSYNC (" + host + ")"})
```

Comment:
lbry.go is a general package used in lots of other places. you can't have a general util function called sendToSlack that says YTSYNC in the message. all ytsync-specific stuff should go into the ytsync package